File: 01/006/main.go
package main

import (
	"fmt"
	"math"
)

func SumDifSquare(num int) int {
	var x, y = 0, 0
	for i := 1; i <= num; i++ {
		x += int(math.Pow(float64(i), 2))
	}
	for i := 0; i <= num; i++ {
		y += i
	}
	return int(math.Pow(float64(y), 2)) - x
}

func main() {
	fmt.Println(SumDifSquare(100))
}

File: 01/006/main.php
<?php
function sumDifSquare($num) {
    $x = array_sum(array_map(fn($a) => $a ** 2, range(1, $num)));
    $y = array_sum(range(1, $num)) ** 2;
    return $y - $x;
}

print sumDifSquare(100);
